src/__tests__/select-test.js

```diff
@@ -0,0 +1,83 @@
+jest.dontMock('../select');
+jest.dontMock('classnames');
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+
+const Select = require('../select').default;
+
+describe('Select', () => {
+    const options = [
+        { value: 'one', label: 'One' },
+        { value: 'two', label: 'Two' },
+        { value: 'three', label: 'Three' }
+    ];
+
+    it('can recieve a default value', () => {
+        const select = TestUtils.renderIntoDocument(<Select options={options} defaultValue={'two'} />);
+
+        expect(select.state.value).toBe('two');
+    });
+
+    it('can recieve a default value', () => {
```

Comment:
Should probably read "does not set value when none is present" or something. Also, testing the use case of passing in a value and checking immediately it would be good.